core/src/main/resources/schema/infinispan-config-15.0.xsd

```diff
@@ -1230,6 +1237,38 @@
               </xs:documentation>
             </xs:annotation>
           </xs:attribute>
+          <xs:attribute name="container" type="xs:boolean" default="${Tracing.container}">
+            <xs:annotation>
+              <xs:documentation>
+                Enable or disable tracing for container category on the given cache.
+                This property can be used to enable or disable tracing at runtime.
+              </xs:documentation>
+            </xs:annotation>
+          </xs:attribute>
+          <xs:attribute name="cluster" type="xs:boolean" default="${Tracing.cluster}">
+            <xs:annotation>
+              <xs:documentation>
+                Enable or disable tracing for cluster category on the given cache.
+                This property can be used to enable or disable tracing at runtime.
+              </xs:documentation>
+            </xs:annotation>
+          </xs:attribute>
+          <xs:attribute name="x-site" type="xs:boolean" default="${Tracing.x-site}">
+            <xs:annotation>
+              <xs:documentation>
+                Enable or disable tracing for x-site category on the given cache.
+                This property can be used to enable or disable tracing at runtime.
+              </xs:documentation>
+            </xs:annotation>
+          </xs:attribute>
+          <xs:attribute name="persistence" type="xs:boolean" default="${Tracing.persistence}">
+            <xs:annotation>
+              <xs:documentation>
+                Enable or disable tracing for persistence category on the given cache.
+                This property can be used to enable or disable tracing at runtime.
```

Comment:
This needs an update? The persistence is verified only during initialization.

Reply:
going to fix it... but yes it is supposed to be changed at runtime

Reply:
the persistence is supposed to be updatable at runtime... adding a test

Reply:
Added the test https://github.com/infinispan/infinispan/pull/11308/commits/15c8ee3c8509f7f2f6e72e37626bae9943586d99